general/development/process/release/index.md

```diff
@@ -19,13 +19,12 @@ sidebar_position: 7
 
 ## 7 weeks prior
 
-| # | Major | Minor | Task | Responsibility |
-| :---: | :---: | :---: | --- | --- |
+| # | Major | Minor | Task                                                                                                                                                                                                                                                                                                                                                                                                 | Responsibility |
```

Comment:
This change is not required (it was working properly before it was done). Is there any reason for adding all these lines?